portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +31,31 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Fareeda!</h1>");
+
+    ArrayList<String> strings = new ArrayList<String>();
+    strings.add("Hello");
```

Comment:
Did you mean to add different sentences to this list?

Similar to?
```
    quotes = new ArrayList<>();
    quotes.add(
        "A ship in port is safe, but that is not what ships are for. ");
 
  }
````

Can you elaborate more on this?
Also I would recommend defining the list on your strings in a `init` function.

Reply:
I was just putting in random words to test if it was working

Reply:
Sounds good. Make sure to modify it for the final version.